index.bs

```diff
@@ -227,7 +227,7 @@ following steps:
 <ol>
  <li>If |request| is a [=non-subresource request=] for a "sharedworker" or "worker" [=request/destination=], exit without appending any hints to the [=request/header list=].
  <li>Let |hintSet| be an empty [=/client hints set=].
- <li>Run [=retrieve the client hints set=] with |settingsObject|.
+ <li>Run [=retrieve the cached Client Hints set=] with |settingsObject|.
```

Comment:
Do we want to set `hintSet` to the return value (that this currently doesn't return)?

Reply:
(or should it be stored elsewhere? i'm not really sure)

Reply:
We want to rely on the |settingsObject|'s [=environment settings object/client hints set=] here, that's where it's always read from (and temporary modifications can be made).